src/main/java/org/openstreetmap/atlas/geography/atlas/packed/PackedAtlas.java

```diff
@@ -106,11 +126,16 @@
     // Serializer.
     private transient PackedAtlasSerializer serializer;
 
+    // Serialization formats for saving/loading this PackedAtlas
+    private transient AtlasSerializationFormat saveSerializationFormat = AtlasSerializationFormat.JAVA;
+    private transient AtlasSerializationFormat loadSerializationFormat = AtlasSerializationFormat.JAVA;
```

Comment:
Isn't loadSerializationFormat always dependent on the file that it tries to load from? To my mind only the `saveSerializationFormat` would be needed, no?